### DISCUSSION TITLE
Settings for EI and CID methods

### DISCUSSION THREAD

**Discussion by 4psireal2** [qcxms/QCxMS#61] [Q&A] [answered]
Is there a specific guideline on how to set the input parameters for the 2 methods ? I have been using the input scripts from the examples to test a wide range of molecules and I find that the intensity agreement with experimental data somewhat low. I hope to improve this with the input parameters. 

Furthermore, I filter the raw MS spectra so as to remove peaks with negligible intensities for a self-written process of fragment identification by extracting info from the `qcxms.out` file. I'm wondering how other people are implementing this procedure, given that QCxMS and PlotMS haven't included this feature.

**Comment by tobigithub** [comment, marked as answer]
@psireal9 do you have some more info on your general ideas and settings you use? Do you want to investigate both EI and CID spectra? If you talk about wide range of molecules, which ones? No need to fully disclose, but something like MW200-500Da with high number of rotatable bonds, or maybe the compound classes. Also how many compounds, 100s, 1000s, 10,000s? 

The parameters and DOCs are well described https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms_run.html but of course not every keyword combination is listed. That is something to figure out in benchmarks and your own parameter test sets across a wide range of different conditions. Basically use candidates with the worst similarity matches and then start a parameter optimization (design of experiment or DOE) across all valid parameters. Then check if the parameter settings that gave the largest improvements in MS or MS/MS similarity scores can be universally applied to other compounds and compound classes. Best way is to start with very small molecules and then increase to larger MW ranges. We have done that in the past with limited success on a couple of thousand compounds. 

For **CID**, this paper is a good starting point covering around 327 unique molecules, but the QCxMS standard settings were not changed:
**Benchmarking tandem mass spectra of small natural product molecules via ab initio molecular dynamics**
https://chemrxiv.org/engage/chemrxiv/article-details/64210837647e3dca998ead86

For **EI** data, we have done a number of parameter optimizations in the past, for example this one:
**Predicting in silico electron ionization mass spectra using quantum chemistry**
https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00470-3

Scanning the general QCxMS publication line could be helpful too.
https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=qcxms&btnG=

Regarding the post-filter, that is fine either way you do it. In principle that is called a noise filter and is a two or three liner in code, such as: if peak intensity <5% remove peaks. You could show that on synthetic or existing data or GNPS data with different percentages. I am not sure how it much it will increase similarity scores unless you can show that on a couple of hundred examples. If that is true, its probably worth to post it here or publish a paper about it.
